templates/web/base/report/_inspect.html

```diff
@@ -43,11 +43,11 @@
           [% cat_prefix = category | lower | replace('[^a-z]', '') %]
           [% cat_prefix = "category_" _ cat_prefix _ "_" %]
           [% IF category == problem.category %]
-            <p data-category="[% category | html %]" data-priorities="[% priorities_by_category.$category %]">
+            <p data-category="[% category | html %]" data-priorities='[% priorities_by_category.$category %]' data-defect-types='[% category_defect_types.$category %]'>
```

Comment:
What if one of the defect types of priorities contains an apostrophe? I don't think JSON encoded will escape apostrophes?